value/src/main/java/com/google/auto/value/processor/autovalue.vm

```diff
@@ -201,7 +204,7 @@ final class $subclass$formalTypes extends $origClass$actualTypes {
   #if ($builderIsInterface) implements #else extends #end
       ${builderTypeName}${builderActualTypes} {
 
-    private final $bitSet set$ = new ${bitSet}(${props.size()});
+    private final $bitSet set$ = new ${bitSet}();
```

Comment:
What's the motivation in removing the count initialization rather than using the non-nullable property count?